### PR TITLE
Update/requested changes

### PR DESCRIPTION
**Pull request by cqc-melf**

### CODE REVIEW

File: examples/python/qir_introduction.py [hunk 1]
```diff
@@ -2,16 +2,16 @@
 
 # # QIR generation from a pytket circuit
```
Comment (cqc-alec) [inline examples/python/qir_introduction.py]: Weren't we going to delete this example completely?

File: pytket/qir/conversion/api.py [hunk 2]
```diff
@@ -37,29 +37,27 @@ class ReturnTypeQIR(Enum):
 def pytket_to_qir(
     circ: Circuit,
     name: str = "Generated from input pytket circuit",
-    returntype: ReturnTypeQIR = ReturnTypeQIR.BINARY,
+    returntype: QIRFormat = QIRFormat.BINARY,
```
Comment (cqc-alec) [inline pytket/qir/conversion/api.py]: How about renaming `returntype` to `format`?

File: examples/python/qir_introduction.py (hunk 1 above)
Comment (cqc-melf) [reply examples/python/qir_introduction.py]: done in @cqc-melf
[remove example](https://github.com/CQCL/pytket-qir/pull/36/commits/ff0b02a3cf9e045cfaba2351798f33fc575f81e0)
[ff0b02a](https://github.com/CQCL/pytket-qir/pull/36/commits/ff0b02a3cf9e045cfaba2351798f33fc575f81e0)

Comment (cqc-melf) [reply examples/python/qir_introduction.py]: done in @cqc-melf
[remove example](https://github.com/CQCL/pytket-qir/pull/36/commits/ff0b02a3cf9e045cfaba2351798f33fc575f81e0)
[ff0b02a](https://github.com/CQCL/pytket-qir/pull/36/commits/ff0b02a3cf9e045cfaba2351798f33fc575f81e0)

File: pytket/qir/conversion/api.py (hunk 2 above)
Comment (cqc-melf) [reply pytket/qir/conversion/api.py]: done in @cqc-melf
[rename format parameter](https://github.com/CQCL/pytket-qir/pull/36/commits/2357a71e06280cf9b1dbaff53fb2acfd393f0866)
[2357a71](https://github.com/CQCL/pytket-qir/pull/36/commits/2357a71e06280cf9b1dbaff53fb2acfd393f0866)

Comment (cqc-melf) [reply pytket/qir/conversion/api.py]: pylint does not like that, would do qir_format instead? (done in @cqc-melf
[update format parameter name](https://github.com/CQCL/pytket-qir/pull/36/commits/a1e7c6e15651d02258f6607a31c7b45f70a7f293)
[a1e7c6e](https://github.com/CQCL/pytket-qir/pull/36/commits/a1e7c6e15651d02258f6607a31c7b45f70a7f293) )

File: pytket/qir/conversion/api.py [hunk 3]
```diff
@@ -37,29 +37,27 @@ class ReturnTypeQIR(Enum):
...
 ) -> Union[str, bytes, None]:
     """converts given pytket circuit to qir
     :param circ: given circuit
     :type circ: pytket circuit
     :param name: name for the qir module created
     :type name: str
-    :param returntype: format of the generated qir, defaut value is binary
-    :type returntype: ReturnTypeQIR
+    :param format: format of the generated qir, default value is binary
+    :type format: QIRFormat
```
Comment (cqc-alec) [inline pytket/qir/conversion/api.py]:
```suggestion
    :param qir_format: format of the generated qir, default value is binary
    :type qir_format: QIRFormat
```

**Review by cqc-alec — CHANGES_REQUESTED** [review]

**Review by cqc-alec — APPROVED** [review]

**Pull request merged**